pkg/controller/flink/config.go

```diff
@@ -17,7 +18,7 @@ const (
 	BlobDefaultPort                = 6125
 	UIDefaultPort                  = 8081
 	MetricsQueryDefaultPort        = 50101
-	OffHeapMemoryDefaultFraction   = 0.5
+	SystemMemoryDefaultFraction    = 0.5
```

Comment:
I think this should be lowered, since the new config includes off-heap. Maybe set this to 0.2 by default (while keeping the existing default for off-heap)?